crates/turborepo-api-client/src/lib.rs

```diff
@@ -213,6 +224,37 @@ impl APIClient {
         })
     }
 
+    pub async fn get_spaces(&self, token: &str, team_id: Option<&str>) -> Result<SpacesResponse> {
+        // create url with teamId if provided
+        let endpoint = match team_id {
+            Some(team_id) => format!("/v0/spaces?limit=100&teamId={}", team_id),
+            None => "/v0/spaces?limit=100".to_string(),
```

Comment:
Do we need to limit=100? Shouldn't there be only 1?

Reply:
There will be as many as you currently belong to! In the future we can filter by repo url